osu.Game/Graphics/UserInterfaceV2/OsuDirectorySelectorHiddenToggle.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Cursor;
+using osu.Framework.Localisation;
+using osu.Game.Graphics.UserInterface;
+using osuTK.Graphics;
+
+namespace osu.Game.Graphics.UserInterfaceV2
+{
+    internal class OsuDirectorySelectorHiddenToggle : OsuCheckbox, IHasTooltip
+    {
+        public LocalisableString TooltipText => @"Show hidden items";
```

Comment:
Not 100% sure about displaying tooltip rather than a label, but should at least update based on current state (i.e. display "Hide hidden items" when toggled on)

Reply:
I've removed the tooltip in favor of a checkbox label. The text feels cramped though, and breadcrumb directories get layered behind the checkbox if the breadcrumb display is long enough (though that's a separate issue). Maybe this should be a button like the basic implementation?

Reply:
It will have to show its current state, so a button like basic implementation won't work well (and could've used `BasicCheckbox`, actually). The breadcrumb overflow concern is somewhat out of scope and tracked in https://github.com/ppy/osu-framework/issues/4704, but could try making the label shorter I suppose?

Reply:
Put it in its own row, not inline with the breadcrumb row.